### ISSUE TITLE
fetch balance on CEX bug

### ISSUE THREAD

**Issue by ivicaca** [ccxt/ccxt#754]
After updating the ccxt library from 1.10.97 (or something similar) to 1.10.282 the following bug appears after invoking the function `fetchBalance`():

...
  File "C:\Program Files\Python27\lib\site-packages\ccxt\cex.py", line 126, in fetch_balance
    'free': float(balances[currency]['available']),
TypeError: string indices must be integers

**Comment by kroitor** [comment]
Can you please show verbose output?

**Comment by ivicaca** [comment]
The method where error appears is

```
    def fetch_balance(self, params={}):
        self.load_markets()
        balances = self.privatePostBalance()
        result = {'info': balances}
        currencies = list(balances.keys())
        for i in range(0, len(currencies)):
            currency = currencies[i]
            if currency in balances:
                account = {
                    'free': float(balances[currency]['available']),
                    'used': float(balances[currency]['orders']),
                    'total': 0.0,
                }
                account['total'] = self.sum(account['free'], account['used'])
                result[currency] = account
        return self.parse_balance(result)
```

while debugging, the `balances `value is:
{u'username': u'up105237066', u'ZEC': {u'available': u'14.06861920', u'orders': u'0.95552393'}, u'USD': {u'available': u'0.14', u'orders': u'0.00'}, u'BCH': {u'available': u'0.01841842', u'orders': u'0.00000000'}, ...

currencies list value is:
<type 'list'>: [u'username', u'ZEC', u'USD', u'BCH', u'GBP', u'GHS', u'RUB', u'timestamp', u'DASH', u'BTG', u'BTC', u'ETH', u'EUR']

it seems that `username` does not belong here.

**Comment by kroitor** [comment]
Thanks for reporting, will fix it shortly!
